client/ca.pem

```diff
@@ -0,0 +1,32 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
maybe move to "cert directory" or something for "testdata"

Reply:
Done.